package-builder/flox-build.mk

```
@@ -245,12 +245,14 @@ define BUILD_local_template =
 	        $(_t3) $($(_pvarname)_logfile) -- $(_bash) -e $($(_pvarname)_buildScript)
 	$(_V_) $(_nix) build -L --file $(_libexec_dir)/build-manifest.nix \
 	  --argstr name "$(_name)" \
-	  --argstr flox-env "$(FLOX_ENV)" \
-	  --argstr build-wrapper-env "$(_build_wrapper_env)" \
-	  --argstr install-prefix "$(_out)" \
-	  --argstr nixpkgs-url "$(BUILDTIME_NIXPKGS_URL)" \
-	  --out-link "result-$(_pname)" \
-	  '^*'
+      --argstr pname "$(_pname)" \
+      --argstr version "$(_version)" \
+      --argstr flox-env "$(FLOX_ENV)" \
+      --argstr build-wrapper-env "$(_build_wrapper_env)" \
+      --argstr install-prefix "$(_out)" \
+      --argstr nixpkgs-url "$(BUILDTIME_NIXPKGS_URL)" \
+      --out-link "result-$(_pname)" \
+      '^*'
```

notice that _tabs_ cahnged to _spaces_ , `make` might be whitespace sensitive here, or did you change the indentation on purpose?

```suggestion
      --argstr pname "$(_pname)" \
      --argstr version "$(_version)" \
      --argstr flox-env "$(FLOX_ENV)" \
      --argstr build-wrapper-env "$(_build_wrapper_env)" \
      --argstr install-prefix "$(_out)" \
      --argstr nixpkgs-url "$(BUILDTIME_NIXPKGS_URL)" \
      --out-link "result-$(_pname)" \
      '^*'
```

---

Definitely didn't change it on purpose.

---

Fixed